deployment/cdk/app.py

```diff
@@ -135,14 +137,26 @@ def __init__(
         **kwargs: Any,
     ) -> None:
         """Define stack."""
-        super().__init__(scope, stack_id, env=env, *kwargs)
+        super().__init__(scope, stack_id, env=env, stack_name=stack_name, **kwargs)
+
+        if settings.permissions_boundary_name is not None:
+            boundary = iam.ManagedPolicy.from_managed_policy_name(
+                self,
+                'Boundary',
+                settings.permissions_boundary_name
+            )
+            iam.PermissionsBoundary.of(self).apply(boundary)
+
+        if settings.vpc_id is not None:
+            vpc = ec2.Vpc.from_lookup(self, 'VPC', vpc_id=settings.vpc_id)
+        else:
+            vpc = ec2.Vpc(self, f"{stack_name}-vpc", max_azs=2)
```

Comment:
Any reason we're limiting this to 2 availability zones?

Reply:
0b47c104c7e5a3a79bc71acbf3c006d0ff897155